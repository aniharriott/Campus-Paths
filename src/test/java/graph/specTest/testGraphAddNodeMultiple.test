# Test Graph addNode Multiple file.
# Tests that multiple nodes can be
# added to one graph.

# Create graph
CreateGraph graph1

# Add multiple nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3