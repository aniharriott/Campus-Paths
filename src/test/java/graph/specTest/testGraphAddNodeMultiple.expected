# Test Graph addNode Multiple file.
# Tests that multiple nodes can be
# added to one graph.

# Create graph
created graph graph1

# Add multiple nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph3